blog(embertimes103): Visual Regression Video (#160)
ember-learn/ember-blog: source/2019-06-21-the-ember-times-issue-103.md

  
---

## [Talk Video: Visual Regression Testing Made Easy 📺](https://www.youtube.com/watch?v=d0ZCJNeybbs)

> [@patocallaghan](www.github.com/patocallaghan) gave a lightning talk on “Visual Regression Testing Made Easy” at the April 2019 [DublinJS Meetup](https://www.meetup.com/DublinJS/). In his talk, Pat used [Percy](https://percy.io/) to scale testing visual regressions from manually doing them to having it automated. This was useful to us by picking up potential CSS bugs, removing certain types of tests, and making large refactoring and migrations easier since we can check the live result rather easily. “Styles can sometimes make bad things happen,” Percy helps give you confidence when you make big changes.

Reviewer: Below, I changed the pronoun to you, to keep it consistent with the rest of the paragraph, and made the sentence shorter to get to the point. Would you be okay with this change?

```suggestion
[@patocallaghan](https://github.com/patocallaghan) gave a lightning talk on “Visual Regression Testing Made Easy” at the April 2019 [DublinJS Meetup](https://www.meetup.com/DublinJS/). In his talk, Pat used [Percy](https://percy.io/) to scale testing visual regressions from manually doing them to having it automated. You can detect potential CSS bugs, remove certain types of tests, and make large refactoring and migrations. “Styles can sometimes make bad things happen.” Percy gives you confidence when you make big changes.
```
